app/public/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+
+<html>
+
+	<head>
+		 <script src="/jquery/jquery.js"></script>
+
+	</head>
+
+	
+	<body>
+		<menu></menu>
+	
+		<div class="container">
+			<h1 class="text-primary">Hello Chingus</h1>
+			<div class="p-3 mb-2 bg-success text-white">
+				<h1 class="display-4">Glorious Pints</h1>	
+			</div>
+			
+		</div>
+	
+		<footer></footer>
+		        <script> 
+            $(function(){ $("head").load("partials/header.html"); });
```

Comment:
I didn't know that we can add partials to website from client-side JS code like this.